diff --git a/node_modules/@dfinity/agent/lib/cjs/vendor/bls/bls.js b/node_modules/@dfinity/agent/lib/cjs/vendor/bls/bls.js
index 0c899c6..f861be9 100644
--- a/node_modules/@dfinity/agent/lib/cjs/vendor/bls/bls.js
+++ b/node_modules/@dfinity/agent/lib/cjs/vendor/bls/bls.js
@@ -1,4 +1,6 @@
 "use strict";
+import { decode } from 'base-64';
+
 var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
@@ -825,7 +827,16 @@ const wasmBytesBase64 = `
     kLWJ5AwVydXN0Yx0xLjQ5LjAgKGUxODg0YThlMyAyMDIwLTEyLTI5KQZ3YWxydXMGMC4xOC4wDHdhc20tYmluZGdlbhIw
     LjIuNzAgKGI2MzU1YzI3MCk=
 `.replace(/[^0-9a-zA-Z/+]/g, '');
-const wasmBytes = base64Arraybuffer.decode(wasmBytesBase64);
+
+function convertBinaryStringToUint8Array(bStr) {
+        var i, len = bStr.length, u8_array = new Uint8Array(len);
+        for (var i = 0; i < len; i++) {
+            u8_array[i] = bStr.charCodeAt(i);
+        }
+        return u8_array;
+    }
+const wasmBytes = convertBinaryStringToUint8Array(decode(wasmBytesBase64));
+
 /**
  * @returns {number}
  */
